src/main/java/frc/team2412/robot/util/auto/AutonomousChooser.java

```diff
@@ -14,21 +14,32 @@ public interface CommandSupplier {
 
 	public AutonomousChooser() {
 		autonomousModeChooser.setDefaultOption(
-				"LeftLeaveCom", () -> AutonomousTrajectories.getAutoPathByName("LeftLeaveCom"));
+				"FlatLeaveCom", () -> AutonomousTrajectories.getAutoPathByName("LeftLeaveCom"));
 		autonomousModeChooser.addOption(
-				"RightLeaveCom", () -> AutonomousTrajectories.getAutoPathByName("RightLeaveCom"));
+				"BumpLeaveCom", () -> AutonomousTrajectories.getAutoPathByName("RightLeaveCom"));
```

Comment:
Just a note that the names aren't always matching here (as we found out last meeting).